clang/test/Profile/missing-annotations.c

```diff
@@ -0,0 +1,44 @@
+// Test that missing-annotations detects branches that are hot, but not annotated
+
+// test diagnostics are issued when profiling data mis-matches annotations
+// RUN: llvm-profdata merge %S/Inputs/missing-annotations-branch.proftext -o %t.profdata
```

Comment:
Can we avoid creating a new `.proftext` file and instead use `split-file` so that the contents of the test is in one place?
https://llvm.org/docs/TestingGuide.html#extra-files

Reply:
Sure. The use of Inputs here is mostly following the convention I've seen in other tests. I'm unsure how much I can reuse the `.proftext` file between tests, so I can probably just use split-lines for those too.